test/raster.test.js

```diff
@@ -410,9 +410,9 @@ tape('[SPATIAL RESOLUTIONS] Get spatial resolutions / valid spatial resolutions'
   assert.end();
 });
 
-tape('[SPATIAL RESOLUTIONS] Get spatial resolutions / valid spatial resolutions with weight 0.5', function(assert) {
+tape('[SPATIAL RESOLUTIONS] Get spatial resolutions / valid spatial resolutions with weight 0.25', function(assert) {
```

Comment:
I changed the description - it said it was with weight 0.5  but the call below on line 418 was using weight 0.25